2022/day9/Program.fs

```diff
@@ -0,0 +1,91 @@
+﻿// https://adventofcode.com/2022/day/9
+open System.IO
+
+type Pos = struct
+    val X: int
+    val Y: int
+    new (x: int, y: int) = { X = x; Y = y }
+end
+
+// Create a new position by moving from the previous position.
+let movePos (x: int) (y: int) (p: Pos) = Pos(p.X + x, p.Y + y)
```

Comment:
I think here you can take advantage of one of F# strenght type inference: x and y don't need to be annotated as int, the compiler will figure it out. Of course if you want to leave it for clarity, that's perfectly fine!